test/transforms/projectionpursuit.jl

```diff
@@ -0,0 +1,50 @@
+@testset "ProjectionPursuit" begin
+    a = rand(100)
+    b = rand(100)
+    c = b + rand(100)
+    t = Table(; a, b, c)
```

Comment:
Hi @DianaPat can you replace this first example by the example provided in the PairPlots.jl package?

https://github.com/sefffal/PairPlots.jl#examples

Also, can you plot the before and after the transform? Maybe we should add PairPlots.jl as a test dependency to our test/Project.toml? That way we could make use of corner(table) to see the before and after. You can modify the test/Project.toml by activating the test folder and typing `] add PairPlots`